docs/blitz/design/actions/yang.rst

```diff
@@ -100,17 +100,19 @@ subscription, or, you may expect the test to fail (referred to as a negative tes
 .. code-block:: YAML
 
     format:
-      request_mode:    # [STREAM, ONCE, POLL] gNMI subscription mode
-      sub_mode:        # [ON_CHANGE, SAMPLE] gNMI subscription sub_mode
-      encoding:        # [JSON, JSON_IETF, PROTO, ASCII] gNMI val encoding
-      prefix:          # [true | false] gNMI message requires PATH prefix
-      origin:          # [openconfig | rfc7951 | module | <device defined> ] gNMI origin
-      base64:          # [true | false] gNMI set "val" requires Base64 encoding
-      sample_interval: # number of seconds between sampling
-      stream_max:      # seconds to stop stream (default: 0, no max)
-      auto-validate:   # [true | false] automatically validate config messages
-      negative-test:   # [true | false] expecting device to return an error
-      pause:           # pause N seconds between each test (default: 0, no pause)
+      request_mode:     # [STREAM, ONCE, POLL] gNMI subscription mode
+      sub_mode:         # [ON_CHANGE, SAMPLE] gNMI subscription sub_mode
+      encoding:         # [JSON, JSON_IETF, PROTO, ASCII] gNMI val encoding
+      prefix:           # [true | false] gNMI message requires PATH prefix
+      origin:           # [openconfig | rfc7951 | module | <device defined> ] gNMI origin
+      base64:           # [true | false] gNMI set "val" requires Base64 encoding
+      sample_poll:      # number of seconds between sampling
```

Comment:
You are only explaining the sampling in the comment.  Should be something like this:
``# number of seconds between SAMPLE sub_mode or POLL request_mode``